### ISSUE TITLE
[BUG] Unable to access OL6 yum repositories with Vagrant box

### ISSUE THREAD

**Issue by bocekm** [EuroLinux/cloud-images-rfc#3]
**Describe the bug**
I'm unable to access Oracle's OL 6.10 repos when using the [eurolinux-vagrant/oracle-linux-6](https://app.vagrantup.com/eurolinux-vagrant/boxes/oracle-linux-6) box.

**Cloud Image**
Vagrant, libvirt, version 6.10.18

**To Reproduce**

1. Vagrantfile:
```
Vagrant.configure("2") do |config|
  config.vm.box = "eurolinux-vagrant/oracle-linux-6"
end
```
2. `vagrant up`
1. `vagrant ssh`
1. `yum install any_package`
1. See error:
```
Loaded plugins: ulninfo
Setting up Install Process
https://yum.oracle.com/repo/OracleLinux/OL6/UEKR4/x86_64/repodata/repomd.xml: [Errno 14] PYCURL ERROR 77 - "Problem with the SSL CA cert (path? access rights?)"
Trying other mirror.
Error: Cannot retrieve repository metadata (repomd.xml) for repository: public_ol6_UEKR4. Please verify its path and try again
```

**Expected behavior**
I am able to install packages through yum.

**Additional context**
The reason is that `/etc/pki/tls/certs/ca-bundle.crt` is empty. Is it intentional?
```
# rpm -Va ca-certificates
S.5....T.  c /etc/pki/tls/certs/ca-bundle.crt
# ll /etc/pki/tls/certs/ca-bundle.crt
-rw-r--r--. 1 root root 0 Oct  5 19:46 /etc/pki/tls/certs/ca-bundle.crt
```

**Comment by AlexBaranowski** [comment]
Hi!

Thanks for report. I just tested this image and there is problem. We are going to resolve it shortly.

**Comment by AlexBaranowski** [comment]
Ok. So from my research, it seems like:

- Every Enterprise Linux that we support uses the same kickstart. I have no idea why Oracle Linux is not working as expected.
- The problem is present **before** any customization (just after kickstart).
- I added extra tests - checking if https://* sites workes after installation.
- I fixed previous smoke tests because there were gave false negatives.
- I fixed additional tests that we are making on a regular interval with random images.

Now I'm trying to rebuild the image with previous OL iso (we update systems before pushing to Vagrant Cloud so it will have newest packages).

TLDR: We are still working on the issue. 

PS. As the person responsible for this images I'm doing it mostly in my free time, and right now there are protests in my country (I'm actually going on them), so it is slower than I expected :).

**Comment by bocekm** [comment]
Thanks for the information. It's not that urgent, we have a workaround. Good luck with the protests.

**Comment by AlexBaranowski** [comment]
Hi. The newest eurolinux-vagrant/oracle-linux-6   (libvirt, 6.10.19) fixed the issue. Once more thanks for report. Tommorow rest of the providers will be automatically rebuild.

**Comment by bocekm** [comment]
Thanks a lot, Alex.
